### ISSUE TITLE
My implementation in Pytorch doesn't work

### ISSUE THREAD

**Issue by Airpooyan** [richardaecn/class-balanced-loss#2]
Dear authors, thanks for your greate effort to make your code open-source.
I re-implement you **CB-focal Loss**  in Pytorch(Both in your tf-version and my own version), but can't reach the performance reported in your paper.

This is my code. Could you please have a check whether there is something wrong with my code?
**output**: [batch_size, num_classes]
**label**: [batch_size]
**catList**: [num_classes] a list of sampler numbers for each class 

![image](https://user-images.githubusercontent.com/18299981/56867343-b6c99600-6a16-11e9-9b58-ffb82d02ae6e.png)

**Comment by GuoleiSun** [comment]
Hi,

I think your code is fine. I looked it carefully, and I didn't find any problem. Did you reach the performance reported in the paper? Thanks a lot

**Comment by Airpooyan** [comment]
> Hi,
> 
> I think your code is fine. I looked it carefully, and I didn't find any problem. Did you reach the performance reported in the paper? Thanks a lot

I finished my experiments on long-tail CIFAR10 which author released in TF-Record mode but didn't achieved the acc reported in the paper.

**Comment by feidfoe** [comment]
Same issue here as well. (I also used Pytorch for re-implementation)
I hope the authors provide more details and trained parameters for long-tailed CIFAR.

@hbsz123  Please let us know if you have any progress.

**Comment by malarsaravanan1991** [comment]
Quick question :
catList: [num_classes] a list of sampler numbers for each class

sampler number for each class here is calculated for each batch or entire dataset?
